### ISSUE TITLE
Health Check - unable to navigate to additional options via keyboard (Issue 99 from the accessibility list)

### ISSUE THREAD

**Issue by BatJan** [umbraco/Umbraco-CMS#5927]
As described in the issue here https://github.com/umbraco/Umbraco-CMS/issues/5277 it's currently not possible to tab through the list of health checks since they're currently `<div>` elements with a click event attached.

I'll be making a PR for this shortly.
